src/controllers/subject.js

```
@@ -21,6 +21,12 @@ const getSubjectByname = async (req, res) => {
 
   res.status(200).json(subject)
 }
+const getSubjectNamesByCategoryId = async (req, res) => {
+  const { id } = req.params
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)